src/org/jitsi/impl/neomedia/rtp/RTPEncodingDesc.java

```diff
@@ -387,6 +389,16 @@ public boolean matches(byte[] buf, int off, int len)
         return tid == -1 && idx == 0 || tid == temporalId;
     }
 
+    public boolean matches(long ssrc)
+    {
+        if (primarySSRC != ssrc && rtxSSRC != ssrc)
```

Comment:
hmm, looks like either the line for this comment got changed or i accidentally put this on the wrong line.